integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDedicatedMode.java

```diff
@@ -289,6 +326,197 @@ public void testDedicatedModeNoCrd() {
     }
   }
 
+  /**
```

Comment:
Is it possible to add t3channel to the existing MII domain used in the dedicated mode tests and use that to deploy the application using WLST instead of adding new domain? It saves time by not bringing up a new domain.

Reply:
I don't want to modify the common model file used for this image MII_BASIC_IMAGE_NAME + ":" + MII_BASIC_IMAGE_TAG because it is shared by many tests. 
Since anyway this test is running alone, the time it take to run this test shouldn't matter IMO.